src/gmp.c

```diff
@@ -8998,19 +8997,19 @@ results_xml_append_nvt (iterator_t *results, GString *buffer, int cert_loaded)
             assert (0);
           if (!next (&iterator))
             abort ();
+          severity = ovaldef_info_iterator_severity (&iterator);
           buffer_xml_append_printf (buffer,
                                     "<nvt oid=\"%s\">"
                                     "<type>ovaldef</type>"
                                     "<name>%s</name>"
-                                    "<family/>"
+                                    "<cvss_base>%s</cvss_base>"
```

Comment:
Did we agree on adding cvss_base field? ~I suppose it would not be necessary to add it here~ I guess you did add it again for backwards compatibility of report formats. Is that correct?

Reply:
Yes, it's for backward compatibility and to make the nvt elements consistent.